src/library_atomic.js

```diff
@@ -4,7 +4,9 @@
  * SPDX-License-Identifier: MIT
  */
 
+#if ASSERTIONS
 assert(SHARED_MEMORY);
+#endif
```

Comment:
Ditto, I think this a build-time assert so should be fine.

Reply:
Reverted